torch/onnx/__init__.py

```diff
@@ -387,9 +387,7 @@ def wrapper(*args, **kwargs):
             return output
         flat_output_tensors = tuple(
             v.data for v in function._iter_variables(output))
-        assert len(list(function._iter_variables_permissive(
-            list(kwargs.values())))) == 0, \
-            "Passing Variable through kwargs is not supported"
+        # TODO: kwargs aren't traced
```

Comment:
@ezyang what's the correct thing to do here? InstanceNorm passes weight, saved_running_mean, and saved_running_var as kwargs. They used to be Tensors but Tensor and Variable will be the same type.

Reply:
Sigh. Without changing the tracing API, we would need to reconstruct the positional arguments from the keyword arguments, given the argspec of the function, and then pass them on to the tracer. I've written this code once before but it's definitely annoying.